static_src/components/info_logs.jsx

```diff
@@ -15,7 +16,7 @@ export default class InfoLogs extends React.Component {
     return (
       <PanelActions>
         <p>
-          View more logs at <a href="https://logs.cloud.gov">logs.cloud.gov</a> (for East/West environment) or <a href="https://logs.fr.cloud.gov">logs.fr.cloud.gov</a> (for GovCloud environment).
+          View more logs at <a href={ config.platform.logs.url }>{ config.platform.logs.name }</a>.
```

Comment:
I think this would also be addressed by a new skin config design, so I wouldn't touch it. I would like to write something like this:
```javascript
if (config.platform.logs) {
  return <a ... />;
}
// else nothing
```